### PR TITLE
Updated dependencies to fix vulnerabilities

### PR DESCRIPTION
**Pull request by AkhilReddyS-NR**
Updated dependencies to fix vulnerabilities that are found in Trivy scan. This is the [doc](https://docs.google.com/document/d/1P43uzwaSNA6WQ6OMweyohWQVxkmY8troTyRW8I931hs/edit?tab=t.0) that lists all the vulnerabilities that are handled. 
Jira ticket : https://new-relic.atlassian.net/browse/NR-338866

### CODE REVIEW

File: src/requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
-aiohttp==3.9.5 ; python_version >= "3.11" and python_version < "4.0"
+aiohttp==3.10.11 ; python_version >= "3.11" and python_version < "4.0"
 aiosignal==1.3.1 ; python_version >= "3.11" and python_version < "4.0"
 attrs==23.2.0 ; python_version >= "3.11" and python_version < "4.0"
 frozenlist==1.4.1 ; python_version >= "3.11" and python_version < "4.0"
-idna==3.6 ; python_version >= "3.11" and python_version < "4.0"
+idna==3.7 ; python_version >= "3.11" and python_version < "4.0"
 multidict==6.0.5 ; python_version >= "3.11" and python_version < "4.0"
-yarl==1.9.4 ; python_version >= "3.11" and python_version < "4.0"
+yarl>=1.12.0,<2.0 ; python_version >= "3.11" and python_version < "4.0"
```
Comment (maya-jha) [inline src/requirements.txt:7]: Is this syntax correct? How did you determine which version to use?

**Review by maya-jha — COMMENTED** [review]
Have you tested this by deploying to an AWS account?

**Comment by AkhilReddyS-NR** [comment]
> Have you tested this by deploying to an AWS account?

Yes, tested this and it is working fine

File: src/requirements.txt (hunk 1 above)
Comment (AkhilReddyS-NR) [reply src/requirements.txt:7]: Yeah we can use this syntax
Screenshot 2024-11-26 at 12 12 25 PM

**Comment by AkhilReddyS-NR** [comment]
Trivy scan report without any vulnerabilities:
Screenshot 2024-11-26 at 12 16 20 PM

**Review by maya-jha — APPROVED** [review]

**Pull request merged**